.github/workflows/image-builds-release.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and Push for Release
+on:
+  workflow_dispatch:
+    inputs:
+      src_branch:
+        type: string
+        default: 'release-X.Y'
+        description: 'Source branch to build KFP from'
+        required: true
+      target_tag:
+        type: string
+        default: 'X.Y.Z'
+        description: 'Target Image Tag'
+        required: true
+      overwrite_imgs:
+        type: string
+        default: 'true'
+        description: 'Overwrite images in GHCR if they already exist for this tag.'
+        required: true
+      set_latest:
+        type: string
+        default: 'true'
+        description: 'Set latest tag on build images.'
+        required: true
+      add_sha_tag:
+        type: string
+        default: 'true'
+        description: 'Add a sha image tag.'
+        required: false
+jobs:
+  build-images-for-release:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - image: kfp-api-server
+            dockerfile: backend/Dockerfile
+            context: .
+          - image: kfp-frontend
+            dockerfile: frontend/Dockerfile
+            context: .
+          - image: kfp-persistence-agent
+            dockerfile: backend/Dockerfile.persistenceagent
+            context: .
+          - image: kfp-scheduled-workflow-controller
+            dockerfile: backend/Dockerfile.scheduledworkflow
+            context: .
+          - image: kfp-viewer-crd-controller
+            dockerfile: backend/Dockerfile.viewercontroller
+            context: .
+          - image: kfp-visualization-server
+            dockerfile: backend/Dockerfile.visualization
+            context: .
+          - image: kfp-launcher
+            dockerfile: backend/Dockerfile.launcher
+            context: .
+          - image: kfp-driver
+            dockerfile: backend/Dockerfile.driver
+            context: .
+          - image: kfp-cache-deployer
+            dockerfile: backend/src/cache/deployer/Dockerfile
+            context: .
+          - image: kfp-cache-server
+            dockerfile: backend/Dockerfile.cacheserver
+            context: .
+          - image: kfp-metadata-writer
+            dockerfile: backend/metadata_writer/Dockerfile
+            context: .
+          - image: kfp-metadata-envoy
+            dockerfile: third_party/metadata_envoy/Dockerfile
+            context: .
+          - image: kfp-inverse-proxy-agent
+            dockerfile: proxy/Dockerfile
+            context: ./proxy
+    uses: ./.github/workflows/build-and-push.yml
+    with:
```

Comment:
seems like you need the `push` input: https://github.com/kubeflow/pipelines/blob/master/.github/workflows/build-and-push.yml#L98